README.md

```diff
@@ -2,9 +2,9 @@
 
 JavaScript | Stylesheets | Miscellaneous
 --- | --- | ---
-[closure_js_library] | [closure_css_library] | [closure_js_proto_library]
-[closure_js_binary] | [closure_css_binary] | [phantomjs_test]
-[closure_js_test] | | [closure_proto_library] \(Experimental\)
+[closure_js_library] | [closure_css_library] | [phantomjs_test]
```

Comment:
nits: I would put `phantomjs_test` on the same line than `closure_js_test`

Reply:
I don't want to put meaning into rows since it is not necessarily feasible to maintain that.